app/form_objects/support/enable_orders_form.rb

```diff
@@ -7,7 +7,6 @@ class Support::EnableOrdersForm
 
   validates :order_state, inclusion: { in: School.order_states }
   validates :cap, numericality: { only_integer: true, greater_than: 0 }, if: :cap_required?
-  validates_with OrderStateAndCapValidator
```

Comment:
Do we not need to do `validate :cap_lte_allocation` here as well?

Reply:
Yeah, I guess we should - done